### DISCUSSION TITLE
Neogit 1.0.0 is buggy

### DISCUSSION THREAD

**Discussion by ambaradan** [lumen-oss/rocks.nvim#604] [Q&A] [answered]
Version 1.0.0 contains a bug that leaves the buffers it uses open once the operation is closed, this means that after a series of commits you can have four five unnamed buffers open in the session, very annoying. The problem has recently been fixed and the *scm* version fixes the problem so I think it is preferable that *rocks.nvim* installs the scm version by default, this is to avoid headaches for any new users.
Also I can't find any reference on the project repo of version 1.0.0 can anyone point me to it?
Thanks

**Comment by mrcjkb** [comment, marked as answer]
hey 👋 

We provide an option to pin plugins, using `:Rocks pin` and `:Rocks unpin`.
Defaulting to `scm` for certain plugins is a bad idea, because:

- A plugin manager should not contain hacks for individual plugins it manages.
- scm versions are unstable and can't be pinned.

It appears there exists a v2.0.0 tag that was released a month ago.
I'll look into why that wasn't published to luarocks.org later today.

Edit: v2.0.0 is now available on luarocks.org.

**Comment by ambaradan** [comment]
That's what I did, set the pin to the *scm* version and everything works fine. I didn't know there was no option to select the default version to download.
Thank you

**Reply by mrcjkb** [reply]
Just fixed the issue that prevented v2.0.0 from being published.
v2.0.0 of neogit is now available on luarocks.org.
